docker_test.go

```diff
@@ -240,6 +240,14 @@ func TestContainerReturnItsContainerID(t *testing.T) {
 	}
 }
 
+type logger struct {
```

Comment:
NIT: rename to make it clear it's a log consumer
```suggestion
type testLogConsumer struct {
```